Source/Scene/Cesium3DTile.js

```diff
@@ -68,7 +68,15 @@ define([
     "use strict";
 
     /**
-     * DOC_TBA
+     * A tile in a 3D Tiles tileset.  When a tile is first created, its content is not loaded;
+     * the content is loaded on-demand when needed based on the view using
+     * {@link Cesium3DTile#requestContent}.
+     * <p>
+     * Do not construct this directly, instead access tiles through {@link Cesium3DTileset#tileVisible}.
+     * </p>
+     *
+     * @alias Cesium3DTile
+     * @constructor
```

Comment:
Same comment, should it list the parameters? The other classes are missing them too, but maybe that is for a reason?